conjure-go-client/httpclient/response_error_decoder_middleware_test.go

```diff
@@ -176,6 +201,89 @@ func TestErrorDecoderMiddlewares(t *testing.T) {
 	}
 }
 
+func TestGetStatusCodeFromError(t *testing.T) {
```

Comment:
nit: would you mind moving the tests to an `errors_test.go` repo?

Reply:
yeah sure. I forgot they are wrappers.